Gir.toml

```diff
@@ -432,3 +445,7 @@ manual_traits = ["SurfaceExtManual"]
         [[object.function.parameter]]
         name = "region"
         const = true
+    [[object.function]]
+    name = "translate_coordinates"
+    # mutability issue, manual
```

Comment:
`const = true` on the parameter in question doesn't help?

Reply:
Nop it doesn't help :/  I keep getting
```
   --> src/auto/surface.rs:223:17
    |
223 |                 y,
    |                 ^ expected *-ptr, found `f64`
    |
    = note: expected raw pointer `*mut f64`
                      found type `f64`

 ```

Reply:
Has wrong annotations in C: the parameters should be `(inout)`, then this should work here.

Reply:
Please send MRs upstream for all these annotation problems and document that accordingly in the `Gir.toml` when implementing it manually :) Here the function should probably be `fn (self, x, y) -> (f64, f64)`

Reply:
I don't think it's needed here? as the function returns a boolean per the docs and it's just a mutability 
```
gboolean gdk_surface_translate_coordinates (GdkSurface *from,
                                            GdkSurface *to,
                                            double     *x,
                                            double     *y);
```

Reply:
Why shouldn't it be needed? gobject-introspection can't know otherwise that `x` and `y` are `inout` parameters. You need to pass the coordinates in "from" space there, and get back the coordinates in "to" space